docs/UserGuide.adoc

```diff
@@ -18,7 +18,12 @@ By: `Team W17-2`      Since: `Sep 2018`      Licence: `MIT`
 
 == Introduction
 
-ModUNI is an application for NUS students who want to quickly generate a plan for the modules to take during their entire university life. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+ModUNI is an application for NUS students who want to quickly generate a plan for the modules to take during their entire university life. +
```

Comment:
Are we using "modsUni" or "ModUNI"? AboutUs.adoc uses modsUni. Might want to standardise that. @CS2103-AY1819S1-W17-2/developers

Reply:
I suggest using modsUni.